engine/lite/embeddedDeploymentManager/src/main/scala/pl/touk/nussknacker/engine/embedded/RequestResponseAkkaHttpHandler.scala

```diff
@@ -0,0 +1,98 @@
+package pl.touk.nussknacker.engine.embedded
```

Comment:
It doesn't look like embedded logic - it will be used also in runtime image, won't be?

Reply:
:+1: